File: Java/Iniciante/1017 - Gasto de Combustível.java
/*
* Autor: Pierre Vieira
* Data de submissão: 11/02/2020 22:13:41
*/
import java.util.Scanner;
public class Main {

    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int tempo, velocidade, distancia;
        float litros;
        tempo = teclado.nextInt();
        velocidade = teclado.nextInt();
        distancia = velocidade * tempo;
        litros = (float) distancia / 12;
        System.out.printf("%.3f\n", litros);
    }
}

File: C#/Iniciante/1017 - Gasto de Combustível.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 12:59:08
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int tempo, velocidade_media;
            tempo = int.Parse(Console.ReadLine());
            velocidade_media = int.Parse(Console.ReadLine());
            double distancia = (double) tempo * velocidade_media;
            Console.WriteLine((distancia/12).ToString("F3", CultureInfo.InvariantCulture));
        }
    }
}
